### PR TITLE
Apply tag and location api

### PR DESCRIPTION
**Pull request by servinlp**

### CODE REVIEW

File: routes/api.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ router.get( '/tags', async ( req, res ) => {
 
 	res.json( {
 		succes: true,
-		tags: tag
+		data: tag
 	} )
```
Comment (moniac) [inline routes/api.js]: this is a good choice

File: views/components/_base/_libs.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.autocomplete {
+	position: relative;
+
+	ul {
+		position: absolute;
+		bottom: 0;
+		left: 0;
```
Comment (moniac) [inline views/components/_base/_libs.scss]: unitless is smart

File: views/components/lib/autocomplete.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+	autocomplete.forEach( el => {
+
+		el.addEventListener( 'input', autocompleteFromApi )
+
+	} )
+
+}
+
+async function autocompleteFromApi( e ) {
+
```
Comment (moniac) [inline views/components/lib/autocomplete.js]: i love async

File: views/index.scss [hunk 4]
```diff
@@ -2,6 +2,8 @@
 @import './components/_base/_mixins.scss';
 @import './components/_base/_variables.scss';
 @import './components/_base/_base.scss';
+@import './components/_base/_libs.scss';
+
```
Comment (moniac) [inline views/index.scss]: this is smart

**Comment by servinlp** [comment]
Thanks @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac @moniac

**Pull request merged**